File: Week 1/hello.js
const readline = require("readline");

// //output
// console.log("Hello World");

//input
//create readline interface
rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

//output question
rl.question("What is your name? ", (name)=>{
  rl.question("What is your favorite color? ", (color)=>{
    console.log(`${name}'s favorite color is ${color}`);
    rl.close();
  });
});

//exit process
rl.on("close", ()=>{
  console.log("bye");
  process.exit();
});

File: Week 1/hello.py
import sys

#output
print("Hello World");

#input
name = input("What is your name?");
print("Your name is " +name);

#output to file
textfile = open("pythonOut.txt", "w");
textfile.write("Last updated by " + name);
textfile.close();

#read from file
textFileR = open("pythonOut.txt", "r");
print(textFileR.read());
textFileR.close();